src/components/sbb-datepicker-toggle/sbb-datepicker-toggle.tsx

```diff
@@ -177,44 +175,48 @@ export class SbbDatepickerToggle implements ComponentInterface {
     return undefined;
   }
 
-  public render(): JSX.Element {
-    return (
-      <Host slot="prefix">
-        <sbb-tooltip-trigger
-          aria-label={i18nShowCalendar[this._currentLanguage]}
-          iconName="calendar-small"
-          disabled={!this._datePickerElement || this._disabled}
-          negative={this.negative}
-          ref={(trigger) => {
-            this._triggerElement = trigger;
+  protected override render(): TemplateResult {
+    setAttribute(this, 'slot', 'prefix');
+    setAttributes(this._triggerElement as HTMLElement, {
+      'aria-label': i18nShowCalendar[this._currentLanguage],
+      'icon-name': 'calendar-small',
+      disabled: !this._datePickerElement || this._disabled,
+      negative: this.negative,
+      'data-icon-small': true,
+    });
+    return html`
+      ${this._triggerElement}
```

Comment:
Why not do it the usual way? 
You had problems with `.trigger=${this._triggerElement}`? (line 195)
I fear this might interfere with the canonic render process

Reply:
Using ref either causes loops or straight does not work, id reference is not queryable inside the shadowRoot this is the only way I was able to make it work